From 185128cf8af7e05ccae8185425c8073809fb8810 Mon Sep 17 00:00:00 2001
From: Michael Heimpold <mhei@heimpold.de>
Date: Sat, 3 Mar 2018 14:32:38 +0100
Subject: [PATCH 10/15] Fix typo in CMakeLists.txt

This option enables the XML backend, not the serial backend.
Looks like a typical C&P typo :-)

Signed-off-by: Michael Heimpold <mhei@heimpold.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e3ed168..e697ede 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -249,7 +249,7 @@ else()
 endif()
 
 if (LIBXML2_FOUND)
-	option(WITH_XML_BACKEND "Enable the serial backend" ON)
+	option(WITH_XML_BACKEND "Enable the XML backend" ON)
 
 	if (WITH_XML_BACKEND)
 		set(LIBIIO_CFILES ${LIBIIO_CFILES} xml.c)
-- 
2.11.0

